src/tokenizer.py

```diff
@@ -9,15 +9,15 @@
 from enum import Enum
 from pathlib import Path
 
-import src.scripts.preprocessing.kutuzov.rus_preprocessing_udpipe as kutuzov
+import scripts.preprocessing.kutuzov.rus_preprocessing_udpipe as kutuzov
 
 # Set up the configuration
 path_current_directory = os.path.dirname(__file__)
 path_config_file = os.path.join(path_current_directory, '../',
                                 'config.ini')
 config = configparser.ConfigParser()
 config.read(path_config_file)
-models_dir = config['PATHS']['models_dir']
+treetagger_dir = config['PATHS']['treetagger_dir']
```

Comment:
`models_dir` holds more than just treetagger; it holds other preprocessing models (udpipe) and the word embeddings models after training. How should we name this config to allow for everything? Maybe a 'tokenizer_dir' and 'trained_models' dir or something of the sort?